src/main/java/edu/rosehulman/covidtracer/model/QuarantineLocation.java

```diff
@@ -33,6 +34,8 @@
 
 	@Column(name = "isClean", nullable = false)
 	private boolean isClean = true;
+	
+	public QuarantineLocation() {}
```

Comment:
What does this code do?

Reply:
It is needed by Spring in order to generate the objects at startup. This is how spring verifies/builds these entities in the database.

Reply:
It is needed by Spring in order to generate the objects at startup. This is how spring verifies/builds these entities in the database.

Reply:
But are we using this particular constructor anywhere in the code?

Reply:
Or was it autogenerated and do we need to remove it?

Reply:
For whatever reason, Spring gets mad if you don't have it. Don't the other classes have blank constructors in them?

Reply:
Yeah they do, I apologize, didn't notice that was a feature of spring